testing/tests/DevExpress.knockout/textBox.tests.js

```diff
@@ -0,0 +1,29 @@
+const $ = require('jquery');
+const ko = require('knockout');
```

Comment:
Let's rewrite `require` to `import`

Reply:
Was updated here: [83e8e4e131d792882cdedfb3a6dadb0eb4b88383]. Check please.